hw2/04-background.js

```diff
@@ -0,0 +1,59 @@
+// Add your code here
+window.addEventListener('load', (event) => {
```

Comment:
Random note: It feels a little odd that the background color is changing when the site loads, but the button says "Start" and when you click it, it doesn't stop. Thus it requires 2x clicks to make the color stop. Maybe the page should load with the button in a stop state?

Reply:
That's how I interpreted the HW instructions, I thought it was odd too. I think I'll leave it as is for now.